examples/README.md

```diff
@@ -24,6 +24,7 @@ How to try these examples
 Included example folders
 -----------------------
 
+*   **/cloud-discovery** — Includes a code sample that shows how to use Hazelcast.Cloud
```

Comment:
I think a format should be chosen and use for all of them. I see `includes ..` `contains..` `code samples ..`.